src/components/edit.jsx

```diff
@@ -0,0 +1,254 @@
+import { useSelector } from "react-redux"
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import { Button } from "./Button";
+import MenuItem from '@mui/material/MenuItem';
+import { updateWebsite } from "../api/api";
+import './edit.css'
+
+export const Edit = () => {
+
+    let cpu = [
```

Comment:
The values shouln't be listed here, and they should be equal to the values from the server.

Reply:
אין כזו פונקציה בצד שרת אולי כשיהיה נשנה ?אחרת זה לא יעבוד?

Reply:
אני שמה בנתיים את רשימת הCPU בצד שרת שלנו ומקבלת אותם ממנו מקוה שזה בסדר